xmtp-inbox-ios/Views/MessageComposerView.swift

```diff
@@ -82,21 +80,17 @@ struct MessageComposerView: View {
 		}
 		.padding(.horizontal, 8)
 		.overlay(RoundedCorner(radius: 16, corners: [.topLeft, .topRight, .bottomLeft]).stroke(Color.actionPrimary, lineWidth: 2))
-		.disabled(isSending)
```

Comment:
Removed this because it was causing the textfield to lose focus which would hide the keyboard briefly, then have it bounce back up after sending went through.